config/clusters/berkeley-geojupyter/common.values.yaml

```diff
@@ -0,0 +1,178 @@
+#If the hub is a daskhub, please indent everything under a basehub key
+#basehub:
+nfs:
+  enabled: true
+  pv:
+    enabled: true
+
+    # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+    mountOptions:
+    - rsize=1048576
+    - wsize=1048576
+    - timeo=600
+    - soft   # We pick soft over hard, so NFS lockups don't lead to hung processes
+    - retrans=2
+    - noresvport
+    baseShareName: /
+
+jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: github
+    homepage:
+      templateVars:
+        org:
+          name: berkeley-geojupyter
+          logo_url: https://github.com/user-attachments/assets/0218ad87-061f-4b3a-b1e4-7555a9a83876
+          url: https://geojupyter.org/
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: The Eric & Wendy Schmidt Center for Data Science & Environment at UC Berkeley
+          url: https://dse.berkeley.edu/
+  hub:
+    allowNamedServers: true
+    loadRoles:
+      # Override the user role with scopes for sharing UI & impl
+      user:
+        scopes:
+        - self
+          # Sharing scopes
+        - shares!user
+        - read:users:name
+        - list:users
+
+    config:
+      JupyterHub:
+        authenticator_class: github
+      GitHubOAuthenticator:
+        allowed_organizations:
+        - geojupyter:jupyterhub-users
+      Authenticator:
+        admin_users:
+        - mfisher87
+    extraConfig:
+      # Allow these sharing scopes for oauth tokens issued by the oauth client
+      permitSharingScopes: |
+        c.Spawner.oauth_client_allowed_scopes = [
```

Comment:
This can go under `hub.config.Spawner.oauth_client_allowed_scopes` instead of using custom python.